src/database/queries/delete_events_querie.js

```diff
@@ -0,0 +1,14 @@
+const Events = require('../event_schema');
+
+const deleteQuery = (req, callback) => {
+
+  Events.findByIdAndRemove(req, (err, response) => {
+    if (err) return res.status(500).send(err);
```

Comment:
res is not defined. You don't send anything from inside the query. You can handle the server error using the ```next()``` function in the controller inside the callback.

Reply:
@FarahZaqout @NoureldeanSaed I solved it :+1:

Reply:
@FarahZaqout @NoureldeanSaed  fixed :+1: